doc/source/tutorial/Flower-1-Intro-to-FL-PyTorch.ipynb

```diff
@@ -677,13 +677,13 @@
       "toc_visible": true
     },
     "kernelspec": {
-      "display_name": "Python 3.7.12 64-bit ('flower-3.7.12')",
+      "display_name": "Python 3.7.15 64-bit ('flower-3.7.15')",
```

Comment:
I'm not sure how these versions were set in the first place and whether we should update them manually or not. Could you try re-running the notebooks on Google Colab (which recently upgraded to 3.8) and see what you get?

Reply:
Yes indeed, it doesn't change anything when running `!python --version`, it prints `python 3.8.16`. Should I revert the change or not then ?

Reply:
Yes, please revert it for the notebooks